agentic-ai-model-tool-setup/flinksql/README.md

```diff
@@ -0,0 +1,195 @@
+```
+
+Run the plugin from the top-level directory of the `tutorials` repository to create the Confluent Cloud resources needed for this tutorial. Note that you may specify a different cloud provider (`gcp` or `azure`) or region. You can find supported regions in a given cloud provider by running `confluent kafka region list --cloud <CLOUD>`. The plugin should complete in under a minute.
+
+```shell
+confluent quickstart \
+  --environment-name agentic-ai-env \
+  --kafka-cluster-name agentic-ai-cluster \
+  --compute-pool-name agentic-ai-pool
+```
+
+## Create Linear and OpenAI credentials
+
+[Log in](https://auth.openai.com/log-in) to the OpenAI platform. Navigate to the [Project API keys page](https://platform.openai.com/api-keys) and click `Create new secret key`. Save this key because we will need it later when creating a remote model in Flink.
+
+Next, [log in to Linear](https://linear.app/login).
+
+When prompted, give your workspace a unique name and click through the quick start prompts until you get to your workspace home page.
+
+To create a Linear API key, click the workspace dropdown at the top left, then `Settings`. Select `Security & access` in the left-hand navigation, followed by `New API key` under `Personal API keys`. Give the key a name and click `Create`. Save this API key.
+
+## Create Linear and OpenAI connections
+
+Start a Flink SQL shell:
+
+```shell
+confluent flink shell --compute-pool \
+  $(confluent flink compute-pool list -o json | jq -r ".[0].id")
+```
+
+Linear offers both streamable HTTP and SSE-based [MCP endpoints](https://linear.app/docs/mcp#general). Paste your Linear API key into the following statement to create a connection to Linear's streamable HTTP MCP server:
+
+```sql
+CREATE CONNECTION `linear-mcp-connection`
+  WITH (
+    'type' = 'MCP_SERVER',
+    'endpoint' = 'https://mcp.linear.app/mcp',
+    'token' = '<LINEAR_API_KEY>'
+  );
+```
+
+Next, paste your OpenAI API key into the following statement to create a connection to OpenAI's [Chat Completions API](https://platform.openai.com/docs/api-reference/chat/create):
+
+```sql
+CREATE CONNECTION `openai-connection`
+  WITH (
+    'type' = 'openai',
+    'endpoint' = 'https://api.openai.com/v1/chat/completions',
+    'api-key' = '<OPENAI_API_KEY>'
+  );
+```
+
+## Inspect the Linear MCP server
+
+While you can sometimes point a model at an MCP tool directly, it helps to inspect the tool to see which operations are available and what parameters they require.
+
+Run the following command to start an MCP server inspection tool:
+
+```shell
+npx @modelcontextprotocol/inspector@latest https://mcp.linear.app/mcp
+```
+
+In the form on the left, select the `Streamable HTTP` Transport Type, enter `https://mcp.linear.app/mcp` as the URL, and enter your Linear API key as the `Bearer` header.
+
+![MCP inspector](https://raw.githubusercontent.com/confluentinc/tutorials/master/agentic-ai-model-tool-setup/flinksql/img/mcp_inspector.png)
```

Comment:
chicken / egg issue - we need absolute URLs for images to render properly on Confluent Developer, so this won't work until it's merged